Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -1019,7 +1021,7 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "81"
+      - "136"
```

Comment:
change back to the way it was.  Changes in https://github.com/demisto/content/pull/28908

Reply:
resolved